### PR TITLE
Update HashKey Testnet Info

### PR DESCRIPTION
**Pull request by HashKeyCloud**
add rpc info and explorer info

### CODE REVIEW

**Comment by Romek-cloud** [comment]
Sabo

**Review by Romek-cloud — COMMENTED** [review]
przemyslaw sabo

File: _data/chains/eip155-133.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "HashKey Chain Testnet",
   "title": "HashKey Chain Testnet",
   "chain": "HashKey Chain Testnet",
-  "rpc": [],
+  "rpc": ["https://hashkeychain-testnet.alt.technology"],
```
Comment (Romek-cloud) [inline _data/chains/eip155-133.json:5]:
```suggestion
  "rpc":["https://hashkeychain-testnet.alt.technology"],
```

Comment (Romek-cloud) [reply _data/chains/eip155-133.json:5]: Sabo

**Review by ligi — APPROVED** [review]

**Pull request merged**